homeassistant/components/vicare/binary_sensor.py

```diff
@@ -37,24 +37,27 @@ class ViCareBinarySensorEntityDescription(
 
 CIRCUIT_SENSORS: tuple[ViCareBinarySensorEntityDescription, ...] = (
     ViCareBinarySensorEntityDescription(
-        key="circulationpump_active",
-        name="Circulation pump active",
-        device_class=BinarySensorDeviceClass.POWER,
+        key="circulationpump",
```

Comment:
By changing the key you are creating new entities, I don't think that should be the way here so please restore them to the old value

Reply:
```suggestion
        key="circulationpump_active",
```